resources/sass/components/_list_pages.scss

```diff
@@ -17,6 +17,12 @@
   margin-right: 1.5rem;
 }
 
+.oe-list-pages-sort-form .form-select {
+  @media (min-width: 576px) {
```

Comment:
This is not working:
![image](https://github.com/openeuropa/oe_whitelabel/assets/8488617/2f9ffb45-bfd1-49f0-acbc-d9c651df161b)